tests/functional/services/api/conftest.py

```diff
@@ -31,7 +186,7 @@ def remove_image_by_id():
                 # Therefore, I've observed this finalizer running after the account is deleted. It's not the end of
                 # the world, shouldn't be a failed test. If I make this fixture autouse=True, it has been generating an
                 # extra matrix of tests which is worse than just letting the finalizer skip
-                _logger.info("{} account does not exist, ignoring for teardown".format(FT_ACCOUNT))
+                get_logger(__name__).info("{} account does not exist, ignoring for teardown".format(FT_ACCOUNT))
```

Comment:
not sure I follow this change... I believe the previous behavior would be the same.

Reply:
Had to move the get_logger method out of the class, and this was the only method that needed it so just made a minor adjustment

Reply:
It looks like this file had a top-level `_logger` variable that this change removes:

```python
_logger = get_logger(__name__)
```

But you are saying that this was in a class? It is usually common practice to have that `_logger` (or most of the time `logger`) defined at the top so that it can be used anywhere. This line in particular doesn't look like it is using a custom name (e.g. `get_logger('remove_image_by_id')`), so it doesn't look like there is any benefit to  it - the behavior is the same it seems?

Reply:
added it back. it was just throwing errors at the time where I combined the upper level conftest with this one.